pkg/cmd/secret/list/list.go

```diff
@@ -145,8 +149,10 @@ func fmtVisibility(s Secret) string {
 	return ""
 }
 
-func getOrgSecrets(client *api.Client, host, orgName string) ([]*Secret, error) {
-	secrets, err := getSecrets(client, host, fmt.Sprintf("orgs/%s/actions/secrets", orgName))
+func getOrgSecrets(client *api.Client, host, orgName string, limit int) ([]*Secret, error) {
+	path := fmt.Sprintf("orgs/%s/actions/secrets?per_page=%d", orgName, limit)
```

Comment:
This will get us to 100 maximum secrets. To properly manage `per_page` you'll also need to implement paging. Take a look at this code: https://github.com/cli/cli/blob/ca04e2192185e372474624c84b25eab30053f0e5/pkg/cmd/run/shared/shared.go#L198